dashboard/app/controllers/projects_controller.rb

```diff
@@ -537,13 +542,26 @@ def submit
     project = Project.find_by(id: project_id)
     begin
       authorize! :submit, project
-    rescue CanCan::AccessDenied
+    rescue CanCan::AccessDenied => exception
+      Honeybadger.notify(
```

Comment:
The Honeybadger notification feel a bit unnecessary to me (unless we have particular concerns about people unexpectedly hitting this route)? But up to you.

Reply:
Hmmm..I think I'll leave for now. When I circle back to the publishing refactor, I'll revisit. Making a note here! https://codedotorg.atlassian.net/browse/LABS-1185

Reply:
FWIW when I had discussed this with Alice, my understanding was that our system should not allow a user to access this endpoint if they had been previously denied access in the `GET submission_status` action. So hitting this error here means our system has unexpectedly let someone through somehow, and it felt like we should probably know about it (if it does happen)